### PR TITLE
Hpa support

### PR DESCRIPTION
**Pull request by Avi-Robusta**
uses features from PR
https://github.com/robusta-dev/robusta/pull/803
will merge to master after this PR
We list the replias+ events + if there are metrics-server issues we alert the user how to resolve since HPA requires metrics server 
for full metrics server installation
Screen Shot 2023-04-04 at 17 21 41

for a broken apiservice
Screen Shot 2023-04-04 at 17 23 01

### CODE REVIEW

**Comment by arikalon1** [comment]
@Avi-Robusta In all the screenshots desired=running.
Is this how it will look on a real alert? (isn't the alert is when desired != running ?)

**Comment by Avi-Robusta** [comment]
@arikalon1 you are right, it takes like 15 minutes to fire a real alert so I was using simulated ones, here is a real one
Screen Shot 2023-04-09 at 11 28 42

File: playbooks/robusta_playbooks/prometheus_simulation.py [hunk 1]
```diff
@@ -13,6 +13,7 @@ class PrometheusAlertParams(ActionParams):
     :var deployment_name: Deployment name, for a simulated deployment alert.
     :var container_name: Container name, for adding a label on container.
     :var job_name: Job name, for a simulated Job alert.
+    :var hpa: Job name, for a simulated HorizontalPodAutoscaler alert.
```
Comment (RoiGlinik) [inline playbooks/robusta_playbooks/prometheus_simulation.py:16]: typo?

File: src/robusta/core/reporting/consts.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import logging
```
Comment (RoiGlinik) [inline src/robusta/core/reporting/consts.py:1]: I think its slipped here?

File: playbooks/robusta_playbooks/autoscaler.py [hunk 3]
```diff
@@ -1,17 +1,25 @@
+import json
```
Comment (RoiGlinik) [inline playbooks/robusta_playbooks/autoscaler.py:1]: Also not sure this is needed

**Review by RoiGlinik — APPROVED** [review]
Left some minor comments , but can merge after checking them.

File: playbooks/robusta_playbooks/autoscaler.py (hunk 3 above)
Comment (Avi-Robusta) [reply playbooks/robusta_playbooks/autoscaler.py:1]: removed

**Pull request merged**